tests/test_loader.py

```diff
@@ -94,6 +94,30 @@ def test_loading_class():
     assert obj.docstring == "The class docstring."
 
 
+def test_loading_dataclass():
+    loader = Loader()
+    obj = loader.get_object_documentation("tests.fixtures.dataclass.Person")
+    assert obj.docstring == "Simple dataclass for a person's information"
+    assert len(obj.attributes) == 2
+    name_attr = next(attr for attr in obj.attributes if attr.name == "name")
```

Comment:
Nice way of finding an element in a list, didn't know this could be done like this :+1: